### DISCUSSION TITLE
will you support VS 2017?

### DISCUSSION THREAD

**Discussion by dongdong-yang** [Dotnet-Boxed/Templates#156] [Q&A]
cannot find in online templates and cannot run installation too.

**Comment by elexisvenator** [comment]
You can see the progress for vs2017 support here https://github.com/ASP-NET-MVC-Boilerplate/Templates/commits/VS-2017

**Comment by dongdong-yang** [comment]
WOW. Thanks for the great job!  I love it!

**Comment by RehanSaeed** [comment]
Closing as duplicate. Have updated https://github.com/ASP-NET-MVC-Boilerplate/Templates/issues/148 with details.
